pipelines/extension-build.yml

```diff
@@ -18,36 +15,83 @@ pr:
     - /extensions/
     - /pipelines/extension-build.yml
 
-steps:
-- task: NPM@1
-  displayName: npm install
-  inputs:
-    command: install
-    workingDir: extensions
-
-- task: NPM@1
-  displayName: npm run lint
-  inputs:
-    command: custom
-    customCommand: run lint
-    workingDir: extensions
-
-- task: NPM@1
-  displayName: npm run build
-  inputs:
-    command: custom
-    customCommand: run build
-    workingDir: extensions
-
-- task: CopyFiles@2
-  displayName: Stage extension artifact for publish
-  inputs:
-    sourceFolder: extensions
-    contents: '**/*.vsix'
-    targetFolder: $(Build.ArtifactStagingDirectory)
-
-- task: PublishPipelineArtifact@0
-  displayName: Publish extension artifact
-  inputs:
-    artifactName: drop
-    targetPath: $(Build.ArtifactStagingDirectory)
+stages:
+- stage: Build
+  jobs:
+  - job: buildvsix
+    pool:
+      vmImage: 'ubuntu-latest' 
+    steps:
+    - task: NPM@1
+      displayName: npm install
+      inputs:
+        command: install
+        workingDir: extensions
+
+    - task: NPM@1
+      displayName: npm run lint
+      inputs:
+        command: custom
+        customCommand: run lint
+        workingDir: extensions
+
+    - task: NPM@1
+      displayName: npm run build
+      inputs:
+        command: custom
+        customCommand: run build
+        workingDir: extensions
+
+    - task: CopyFiles@2
+      displayName: Stage extension artifact for publish
+      inputs:
+        sourceFolder: extensions
+        contents: '**/*.vsix'
+        targetFolder: $(Build.ArtifactStagingDirectory)
+
+    - task: PublishPipelineArtifact@0
+      displayName: Publish extension artifact
+      inputs:
+        artifactName: drop
+        targetPath: $(Build.ArtifactStagingDirectory)
+
+    - task: ms-devlabs.vsts-developer-tools-build-tasks.tfx-installer-build-task.TfxInstaller@2
```

Comment:
Needs to be fully qualified, or can we use:
```
- task: TfxInstaller@2
```

Reply:
done